code/game/machinery/cryopod.dm

```diff
@@ -0,0 +1,413 @@
+		"<span class='notice'>You climb out of [src]!</span>")
+	open_machine()
+
+/obj/machinery/cryopod/relaymove(mob/user)
+	container_resist_act(user)
+
+/obj/machinery/cryopod/process()
+	if(!occupant)
+		return
+
+	var/mob/living/mob_occupant = occupant
+	if(mob_occupant)
+		// Eject dead people
+		if(mob_occupant.stat == DEAD)
+			open_machine()
+
+		if(!(world.time > despawn_world_time))
+			return
+
+		if(!mob_occupant.client && mob_occupant.stat < 2) // Occupant is living and has no client.
+			if(!control_computer)
+				find_control_computer(urgent = TRUE) // better hope you found it this time
+
+			despawn_occupant()
+
+/obj/machinery/cryopod/proc/handle_objectives()
+	var/mob/living/mob_occupant = occupant
+	// Update any existing objectives involving this mob.
+	for(var/datum/objective/objective in GLOB.objectives)
+		// We don't want revs to get objectives that aren't for heads of staff. Letting
+		// them win or lose based on cryo is silly so we remove the objective.
+		if(istype(objective,/datum/objective/mutiny) && objective.target == mob_occupant.mind)
+			objective.team.objectives -= objective
+			qdel(objective)
+			for(var/datum/mind/mind in objective.team.members)
+				to_chat(mind.current, "<BR><span class='userdanger'>Your target is no longer within reach. Objective removed!</span>")
+				mind.announce_objectives()
+		else if(objective.target && istype(objective.target, /datum/mind))
+			if(objective.target == mob_occupant.mind)
+				var/old_target = objective.target
+				objective.target = null
+				if(!objective)
+					return
+				objective.find_target()
+				if(!objective.target && objective.owner)
+					to_chat(objective.owner.current, "<BR><span class='userdanger'>Your target is no longer within reach. Objective removed!</span>")
+					for(var/datum/antagonist/antag in objective.owner.antag_datums)
+						antag.objectives -= objective
+				if (!objective.team)
+					objective.update_explanation_text()
+					objective.owner.announce_objectives()
+					to_chat(objective.owner.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
+				else
+					var/list/objectivestoupdate
+					for(var/datum/mind/objective_owner in objective.get_owners())
+						to_chat(objective_owner.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
+						for(var/datum/objective/update_target_objective in objective_owner.get_all_objectives())
+							LAZYADD(objectivestoupdate, update_target_objective)
+					objectivestoupdate += objective.team.objectives
+					for(var/datum/objective/update_objective in objectivestoupdate)
+						if(update_objective.target != old_target || !istype(update_objective,objective.type))
+							return
```

Comment:
Are you sure you did not mean `continue`?

Reply:
i didn't write this so no

Reply:
I'm changing this for now because it looks like it *should* be a continue but any other references I find for this are either completely different code or have this as return